modules.yml

```diff
@@ -74,6 +74,8 @@ modules:
   comp/otelcol/converter/impl: default
   comp/otelcol/ddflareextension/def: default
   comp/otelcol/ddflareextension/impl: default
+  comp/otelcol/ddprofilingextension/def: default
+  comp/otelcol/ddprofilingextension/impl: default
```

Comment:
do these also need `used_by_otel: true` ?

Reply:
good call, added in last commit.